circom/contracts/interfaces/IVerifier.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+interface IVerifier {
+    function verify(
+        uint256[2] calldata a,
+        uint256[2][2] calldata b,
+        uint256[2] calldata c,
+        uint256[16] calldata publicOutputs
```

Comment:
This number can't be hardcoded and needs to be generated via tera files (template)